source/MRMesh/MRPython.h

```diff
@@ -127,5 +139,51 @@ struct PythonFunctionAdder
     MRMESH_API PythonFunctionAdder( const std::string& moduleName, PyObject* ( *initFncPointer )( void ) );
 };
 
+template<typename E>
+void throwExceptionFromExpected(E& err)
+{
+    if constexpr (std::is_nothrow_convertible<E, std::string>::value)
+        throw pybind11::value_error(err);
+    else if (std::is_same<E, MR::PathError>::value)
+        switch(err)
+        {
+            case MR::PathError::StartEndNotConnected:
+                throw pybind11::value_error("no path can be found from start to end, because they are not from the same connected component");
+            default:
+                throw std::runtime_error("please, report to the developers for further investigations");
+        }
+    else
+        throw std::runtime_error("please, report to the developers for further investigations");
+}
```

Comment:
1. why sometimes `pybind11::value_error` and sometimes `std::runtime_error`
2. can we have template specializations in place (here just have one for std::sting, and one for `PathError` somewhere near its declaration just cast to string)